library.js

```diff
@@ -0,0 +1,128 @@
+const myLibrary = [];
+
+
+// Book Class
```

Comment:
please remove all the comments in the code. You guys have done a good job of naming the methods appropriately, we don't need comments.